app/services/force/listing_service.rb

```diff
@@ -41,7 +42,10 @@ def self.listings(attrs = {})
 
     def self.raw_listings(opts = {})
       force = opts[:refresh_cache] || false
-      Request.new(parse_response: true).cached_get('/ListingDetails', nil, force)
+      Request.new(parse_response: true)
+             .cached_get('/ListingDetails', { Tenure: 'rental' }, force)
+      Request.new(parse_response: true)
+             .cached_get('/ListingDetails', { Tenure: 'sale' }, force)
```

Comment:
Getting a red line here from code climate, but not 100% sure how test write a test for this function as it's only used by the rake task 🤔